### PR TITLE
Feature/bump nodejs 8.9.4

### PR DESCRIPTION
**Pull request by feedmypixel**
Bump build pack and NodeJs version

### Of Note
[image: ukti/docker-data-hub-base:wip](https://github.com/uktrade/data-hub-frontend/compare/feature/bump-nodejs-8.9.4?expand=1#diff-1d37e48f9ceff6d8030570cd36286a61R94) needs removing before this is merged

Linked [docker-data-hub-base](https://github.com/uktrade/docker-data-hub-base/pull/4) PR

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -91,7 +91,7 @@ aliases:
 
   # Data hub base docker container
   - &docker_data_hub_base
-    image: ukti/docker-data-hub-base
+    image: ukti/docker-data-hub-base:wip
```
Comment (ztolley) [inline .circleci/config.yml]: Can we use a build tagged 8.9.4 or whatever is more appropriate than WIP?

Comment (feedmypixel) [reply .circleci/config.yml]: did consider that but its a throw away branch/automated build and will go and then revert to latest once this work is done with trigger removed

**Comment by feedmypixel** [comment]
> Can you also up the version Dockerfile?

Sure. Does anyone use this anymore? @yeahfro @teneightfive ?

**Comment by feedmypixel** [comment]
> Does any of this need a new yarn.lock file?

`yarn install` doesn't change the `yarn.lock` file. So no this is ok.

**Pull request merged**